extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -112,54 +130,108 @@
                 "firstLevelUrls = re.findall(r'^(?:\\s+$[\\r\\n]+)+(\\- url: [a-zA-Z0-9\\\\.\\s\\-\\/]+$[\\r\\n]+)', result, re.MULTILINE)\n",
                 "urls = re.findall(r'- url: [a-zA-Z0-9\\\\.\\s\\-\\/]+$', result, re.MULTILINE)\n",
                 "headers = re.findall(r'- header: [a-zA-Z0-9\\\\.\\s-]+$', result, re.MULTILINE)\n",
+                "# all the markdown urls are placed at the end of the list. \n",
+                "possibleMarkdowns = re.findall(r'(- url: [a-zA-Z0-9\\\\.\\s\\-\\/]+$[\\r\\n]+)[\\r\\n]', result, re.MULTILINE)\n",
+                "\n",
+                "def getMarkdownFile(url):\n",
+                "    # url are usually defined in toc as: \"- url: <Path\\to\\notebookFile>\"\n",
```

Comment:
Are "usually"? That implies that sometimes they wouldn't be which would break this...is that actually the case? Also I didn't check all the code but you making sure to trim the string - or is this guaranteed not to start with spaces?

Reply:
So currently that's how Jupyter formats the toc.yml unless they change it in the future. Probably mush have written "Currently" instead of "usually"